source/adminguide/virtual_machines.rst

```diff
@@ -325,36 +325,22 @@ To access a VM through the CloudStack UI:
+
+The default format of the internal name is i-<user\_id>-<vm\_id>-<instance.name>, where instance.name is a global parameter. However, If vm.instancename.flag is set to true, and if a name is provided during the creation of a guest VM, the name is appended to the internal name of the guest VM on the host. This makes the internal name format as i-<user\_id>-<vm\_id>-<name>. The default value of vm.instancename.flag is set to false. This feature is intended to make the correlation between instance names and internal names easier in large data center deployments.
+
+The following table explains how a VM name is displayed in different scenarios.
 
 .. cssclass:: table-striped table-bordered table-hover
 
-============================= ======================= ==================== ===================================== ==========================
-User-Provided Display Name    vm.instancename.flag    Hostname on the VM   Name on vCenter                       Internal Name
-============================= ======================= ==================== ===================================== ==========================
-Yes                           True                    Display name         i-<user\_id>-<vm\_id>-displayName     i-<user\_id>-<vm\_id>-displayName
-No                            True                    UUID                 i-<user\_id>-<vm\_id>-<instance.name> i-<user\_id>-<vm\_id>-<instance.name>
-Yes                           False                   Display name         i-<user\_id>-<vm\_id>-<instance.name> i-<user\_id>-<vm\_id>-<instance.name>
-No                            False                   UUID                 i-<user\_id>-<vm\_id>-<instance.name> i-<user\_id>-<vm\_id>-<instance.name>
-============================= ======================= ==================== ===================================== ==========================
+============================= ======================= ====================== ====================== ====================== ===================================== ==========================
+User-Provided Display Name    vm.instancename.flag    User                   Display Name           Hostname on the VM     Name on vCenter                       Internal Name
+============================= ======================= ====================== ====================== ====================== ===================================== ==========================
+Yes                           True                    Name                   Display name           Name                   i-<user\_id>-<vm\_id>-name            i-<user\_id>-<vm\_id>-name
+No                            True                    <instance.name>-<UUID> <instance.name>-<UUID> <instance.name>-<UUID> i-<user\_id>-<vm\_id>-<instance.name> <instance.name>-<UUID>
```

Comment:
@shwstppr can you please review the 2nd case, name on vCenter should be "<instance.name>-<UUID>" and internal name is "i-<user_id>-<vm_id>-<instance.name>"

Reply:
@shwstppr This still needs to be changed, can you please do that.

Reply:
done @harikrishna-patnala